includes/tax-functions.php

```diff
@@ -52,6 +52,11 @@ function edd_get_tax_rate( $country = false, $state = false ) {
 
 	$user_address = edd_get_customer_address();
 
+    $address_line_1 = !empty( $_POST['card_address'] ) ? $_POST['card_address'] : '';
+    $address_line_2 = !empty( $_POST['card_address_2'] ) ? $_POST['card_address_2'] : '';
+    $city           = !empty( $_POST['card_city'] ) ? $_POST['card_city'] : '';
+    $zip            = !empty( $_POST['card_zip'] ) ? $_POST['card_zip'] : '';
```

Comment:
Sorry one more change here @petemolinero totally missed it last time, can we drop a `sanitize_text_field()` around these assignments? Just to be sure we're sanitizing the `$_POST` data before assigning to the variables `$address_line_1, $address_line_2, $city, $zip`?

Reply:
@cklosowski Done!